packages/sanes-chrome-extension/src/extension/bsMessageHandler/internalHandler.ts

```diff
@@ -63,7 +70,20 @@ export async function handleInternalMessage(
           return true;
         };
 
-        signingServer = persona.startSigningServer(getIdentitiesCallback, signAndPostCallback);
+        async function transactionsChangedHandler(transactions: ReadonlyArray<ProcessedTx>): Promise<void> {
```

Comment:
Why this is `async`?

Reply:
Good point. `async` is not necessary anymore after a refactoring